src/hb-buffer.cc

```diff
@@ -1738,7 +1738,7 @@ hb_buffer_add_codepoints (hb_buffer_t          *buffer,
  * @buffer: an #hb_buffer_t.
  * @source: source #hb_buffer_t.
  * @start: start index into source buffer to copy.  Use 0 to copy from start of buffer.
- * @end: end index into source buffer to copy.  Use (unsigned int) -1 to copy to end of buffer.
+ * @end: end index into source buffer to copy.  Use HB_FEATURE_GLOBAL_END to copy to end of buffer.
```

Comment:
One more thing.  The ones in documentation I think should be written as `@HB_FEATURE_GLOBAL_END` for doc linking to work.